lib/recurly/token.js

```diff
@@ -145,5 +145,15 @@ function validate (input) {
     errors.push('last_name');
   }
 
+  if (input.cvv && !this.validate.cvv(input.cvv)) {
+    errors.push('cvv');
+  }
```

Comment:
If cvv is not there, the first condition returns false and 'cvv' will not be added to error. So I think the current implementation is good?